test/utils.go

```diff
@@ -78,6 +79,12 @@ func configure(t *testing.T) config {
 	if !ok {
 		t.Fatal("missing required env var TEST_CREDENTIALS")
 	}
+
+	cfg.policy, ok = os.LookupEnv("TEST_POLICY")
+	if !ok {
+		t.Fatal("missing required env var TEST_POLICY")
```

Comment:
Can we add a description of what this variable should look like?

Reply:
Fixed. I am also thinking about just moving this to the test case so there is no need to specify it. WDYT?